doc/getting-started/building-from-source.rst

```diff
@@ -176,99 +182,51 @@ Assuming that you are at the top level of the acrn-hypervisor directory, perform
 
   .. code-block:: none
 
-    $ make all BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc6cayh.xml \
-    SCENARIO_FILE=$PWD/misc/vm_configs/xmls/config-xmls/nuc6cayh/sdc.xml
+    $ make all BOARD=$PWD/misc/vm_configs/xmls/board-xmls/nuc6cayh.xml \
```

Comment:
The paths used here are no longer valid. This is not a very good example actually as this can also more simply be done using `make all BOARD=nuc6cayh SCENARIO=sdc`.

We should add a more concrete example showing how people can use that with XML files stored **outside** of the repo. I.e. explain that if someone keeps their custom XML files (`BOARD` and `SCENARIO`) in a folder called `my-own-board` that's outside of the `acrn-hypervisor` repo. The example becomes: `make BOARD=../my-own-board/my-board.xml SCENARIO=../my-own-board/my-scenario.xml`

Reply:
Just refined the steps a bit by using arbitrary paths (assumed to be `/path/to/xxx`).